applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -1064,6 +1123,14 @@ app(AppNameOrId) ->
             end
     end.
 
+-spec find_app(kz_term:ne_binary(), kz_term:ne_binary()) -> {'ok', kz_json:object()} |
+                                                            {'error', any()}.
+find_app(Db, Name) ->
+    ViewOptions = [{'key', Name}
+                  ,'include_docs'
+                  ],
+    kz_datamgr:get_single_result(Db, ?CB_APPS_STORE_LIST, ViewOptions).
```

Comment:
This can return `kz_datamgr:data_error() | {'error', 'multiple_results'}` not `{'error', any()}`.